CONTRIBUTING.md

```diff
@@ -19,8 +19,17 @@ When adding a new or updating an existing community member profile be sure to fo
 
 #### Badges for Community Members
 
-Badges are a way to identify projects community members have made contributions to. When a community member successfully completes a project, they get the badge of that community. An example of how a badge is written can be found [here](https://github.com/layer5io/layer5/blob/master/src/collections/members/abishek-kumar/index.mdx), and it will be displayed like [this](https://layer5.io/community/members/abishek-kumar) on the community project.
+Badges are a way to identify projects community members have made contributions to. These badges are assigned to a contributor to acknowledge the contributions made to the respective project. An example of how a badge is written can be found [here](https://github.com/layer5io/layer5/blob/master/src/collections/members/abishek-kumar/index.mdx), and it will be displayed like [this](https://layer5.io/community/members/abishek-kumar) on the community project.
```

Comment:
Good work, @blossom-babs. I'm tossing in a potential tweak for you to consider incorporating.

Reply:
Hello Lee, I didn't consider your initial review before making these edit. I only added what badges constitute off. The above edit for the badges section is much more detailed and I will incorporate that.

Also, what other tweak would you like for me to incorporate?

Reply:
Thank you, @blossom-babs. Regarding a potential follow up PR, a subsequent PR could address questions such as: When should a badge be assigned? What constitutes "engagement" and/or "contribution"? 

These are harder-to-define and will require discussion, though.

Reply:
Yes, I do think that should be discussed, what constitutes engagement and/or contribution

Reply:
Good, good. :thu